Integrations/src/test/java/io/deephaven/integrations/numpy/Java2NumpyCopyTest.java

```diff
@@ -7,9 +7,9 @@
 
 public class Java2NumpyCopyTest extends BaseArrayTestCase {
 
-    final private String file = Configuration.getInstance().getDevRootPath() + "/Integrations/src/test/resources/io/deephaven/integrations/numpy/dh.jpg";
-    final private String q1 = Configuration.getInstance().getDevRootPath() + "/Integrations/src/test/resources/io/deephaven/integrations/numpy/quadrant1.jpg";
-    final private String q2 = Configuration.getInstance().getDevRootPath() + "/Integrations/src/test/resources/io/deephaven/integrations/numpy/quadrant2.png";
+    final private String file = Configuration.getInstance().getDevRootPath() + "/Integrations/src/test/java/io/deephaven/integrations/numpy/dh.jpg";
```

Comment:
consider just io/deephaven/integration/... for each instead, and wherever you consume these paths, use the classloader to load them instead

Reply:
I took a look at this with Alex.  As written, the `DbFile` and `DbFile` impl take `java.io.File` as input, and not `InputStream` which would come from the classloader.  It is not worth a rework to do this change right now.

Reply:
Since this fix is not going in this time, this is ready to close.